### PR TITLE
Updated Readme.md

### PR DESCRIPTION
**Pull request by afro-systems**
- Fixed spelling errors.
- Added systemd options to scripts
- Modified some formatting and sentence structure

### CODE REVIEW

File: plex-argo/README.md [hunk 1]
```diff
@@ -1,40 +1,55 @@
...
     $PlexOnlineToken=(Get-ItemProperty -Path $RegistryPath -Name PlexOnlineToken).PlexOnlineToken
     $PlexOnlineToken='https://plex.tv/api/resources?X-Plex-Token=' + $PlexOnlineToken
     Write-Host 'Plex API URL:' ${PlexOnlineToken}
     (Invoke-WebRequest -UseBasicParsing -Uri ${PlexOnlineToken}).Content
 ```
-# Plex Setup
+
+**Bash** snippet:
+```bash
+ cat /var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Preferences.xml | grep -oP 'PlexOnlineToken="\K[^"]*
```
Comment (danielewood) [inline plex-argo/README.md]: Consider the following:
`grep -oP 'PlexOnlineToken="\K[^"]* '/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Preferences.xml'`

File: plex-argo/README.md [hunk 2]
```diff
@@ -54,6 +70,7 @@ ArgoURL=$(curl -s http://localhost:33400/metrics | grep -oP 'userHostname="https
 
 if [[ $ArgoURL != $PlexAPIcustomConnections ]]; then
     docker container stop plex
+    #systemctl stop plexmediaserver.service
```
Comment (danielewood) [inline plex-argo/README.md]: Lets just make two bash files.
Plex-Argo-DirectoryUpdate-systemd.bash and Plex-Argo-DirectoryUpdate-docker.bash

File: plex-argo/README.md [hunk 3]
```diff
@@ -1,40 +1,55 @@
...
-When you specity a Custom Connection URL in your Plex Media Server, it will publish that URL in the PlexAPI, allowing all your clients to discover alternative paths to your server. This Published information can be seen for your server by going to `https://plex.tv/api/resources?X-Plex-Token=YOUR_API_TOKEN`
-This API token is refreshed every time your Plex Media Server restarts.
 
-You can obtain your current API Token and see your current customConnections URL in the Plex API with the following powershell snippet:
-    
+When you specify a custom connection URL in your Plex Media Server, it will publish that URL in the PlexAPI. This allows all your clients to discover alternative paths to your server. 
+
+This Published information can be seen for your server by going to `https://plex.tv/api/resources?X-Plex-Token=YOUR_API_TOKEN`
+
+**NOTE:** This API token refreshes each time your Plex Media Server restarts.
```
Comment (danielewood) [inline plex-argo/README.md]: refreshes-->changes (since the token itself changes, what it references refreshes)

**Review by danielewood — CHANGES_REQUESTED** [review]
Thanks, added some changes for review.

File: plex-argo/README.md (hunk 1 above)
Comment (afro-systems) [reply plex-argo/README.md]: **Bash** snippet:

- Docker:
```bash
    grep -oP 'PlexOnlineToken="\K[^"]* '/home/ubuntu/plex/config/Library/Application Support/Plex Media Server/Preferences.xml'
```
- systemd:
```bash
    grep -oP 'PlexOnlineToken="\K[^"]* '/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Preferences.xml'
```

**Pull request merged**